README.md

```diff
@@ -4,12 +4,12 @@ A job to approve Pull Requests from [Renovate Bot](https://github.com/renovatebo
 
 For Github, see [renovatebot/renovate-approve-bot](https://github.com/renovatebot/renovate-approve-bot).
 
-## How It Works
+## How it works
 
-On a each run, the bot does:
+On each run, the bot will:
 
 1. Get all the open PRs from the Renovate Bot user
-2. Filter out PRs where "automerge" is not enabled
+2. Filter out PRs where "automerge" is disabled
 3. Approve the "automerge" PRs
```

Comment:
```suggestion
3. Approve all "automerge" PRs, including those previously approved
```

Reply:
> including those previously approved

This is not exactly true, you cannot re-approve an already approved PR, the job will try, but it will get an HTTP 409 "Conflict" response, this will be logged with the exact reason of the error:
https://github.com/maxbrunet/renovate-approve-bot/blob/d82af1696a06b9ee137fc2570ec9840e1d8afbfe/index.js#L96-L102

For simplicity, I would prefer keeping this line as it is, is it good with you?

Reply:
OK